frontend/src/app/shared/components/table-pagination/table-pagination.component.html

```diff
@@ -1,36 +1,36 @@
-<div class="pagination" *ngIf="pagination && isVisible && pagination.total">
-  <nav class="pagination--pages">
-    <ul class="pagination--items">
+<div class="op-pagination" *ngIf="pagination && isVisible && pagination.total">
+  <nav class="op-pagination--pages">
+    <ul class="op-pagination--items">
 
-      <li [hidden]="pagination.page == 1 || !showPageSelections" class="pagination--item">
+      <li [hidden]="pagination.page == 1 || !showPageSelections" class="op-pagination--item">
         <button
           type="button"
           (click)="showPage(pagination.page - 1)"
           rel="prev start"
           [attr.aria-label]="text.label_previous"
-          id="pagination--prev-link"
+          id="op-pagination--prev-link"
         >
           &#60;
         </button>
       </li>
 
       <ng-container *ngIf="prePageNumbers.length > 0">
         <li *ngFor="let pageNumber of prePageNumbers"
-            class="pagination--item">
+            class="op-pagination--item">
           <button
             type="button"
             (click)="showPage(pageNumber)"
             rel="next"
             [textContent]="pageNumber"
           ></button>
         </li>
-        <li class="pagination--space">…</li>
+        <li class="op-pagination--space">…</li>
```

Comment:
Do we need this? can't the elements just have margins on the side?

Reply:
Yes, we do as the element provide the three dots when the pagination is truncated. The class is the needed to align those correctly with the rest.
<img width="350" alt="Bildschirmfoto 2021-07-07 um 14 48 53" src="https://user-images.githubusercontent.com/7457313/124761940-9d93cc80-df32-11eb-807f-c9c71afdf38a.png">

Reply:
Ahh I see. In that case I'd call this `&--dots` or something similar, but I'm okay with the curent impl